### PR TITLE
Add_product_by_csv_file

### PR DESCRIPTION
**Pull request by longnd-1038**
* Import product to database by csv file
image

### CODE REVIEW

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

File: resources/views/admin/products/add.blade.php [hunk 1]
```diff
@@ -9,6 +9,26 @@
                 </button>
             </div>
 
+            <div class="modal-body" id="attachment-body-content">
+                <form id="edit-form" class="form-horizontal" method="POST" action={{route('products.import')}} enctype="multipart/form-data">
+                    {{csrf_field()}}
+                    <div class="card text-white bg-gradient-info mb-0">
+                        <div class="card-header bg-gradient-info">
+                            <h2 class="m-0">Import Product</h2>
```
Comment (quoctd-0607) [inline resources/views/admin/products/add.blade.php:17]: i18n

File: resources/views/admin/products/add.blade.php [hunk 2]
```diff
@@ -9,6 +9,26 @@
...
+                        </div>
+                        <div class="card-body">
+                            <!-- name -->
+                            <div class="form-group">
+                                <label class="col-form-label" for="modal-input-name">Name</label>
```
Comment (quoctd-0607) [inline resources/views/admin/products/add.blade.php:22]: tương tự

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -1,99 +1,99 @@
...
-            html, body {
-                background-color: #fff;
-                color: #636b6f;
-                font-family: 'Nunito', sans-serif;
-                font-weight: 200;
-                height: 100vh;
-                margin: 0;
-            }
+    <!-- Styles -->
+    <style>
```
Comment (quoctd-0607) [inline resources/views/welcome.blade.php:13]: style viết ra file css. k viết vào blade

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

**Comment by longnd-1038** [comment]
Ready

**Pull request merged**